.editorconfig

```diff
@@ -17,3 +17,7 @@ indent_size = 4
 [std/**.sol]
 indent_style = space
 indent_size = 4
+
+[*.{txt,cmake}]
```

Comment:
Does this propagate to subdirectories?

Reply:
Yes. 

I couldn't figure out however to make it only apply to `CMakeList.txt` though, so it applies to all `.txt` files which probably doesn't do damage.

Reply:
Ok!